app/locales/es/translations.js

```diff
@@ -224,6 +224,7 @@ export default {
     'instructors': 'Instructores',
     'instructorsManageTitle': 'Administrar Instructores',
     'confirmObjectiveRemoval': "¿Está seguro que desea eliminar este objetivo?",
+    'confirmSessionRemoval': "¿Está seguro que desea eliminar este sessión?",
```

Comment:
@dartajax please confirm this translation for `Are you sure you want to delete this session?`